lib/animation.dart

```diff
@@ -101,6 +104,40 @@ class Animation {
     }
   }
 
+  /// Automatically creates an Animation Object using animation data provided by the json file
+  /// provided by Aseprite
+  ///
+  /// [imagePath]: Source of the spritesheet animation
+  /// [dataPath]: Animation's exported data in json format
+  static Future<Animation> fromAsepriteData(String imagePath, String dataPath) async {
+    String content = await rootBundle.loadString(dataPath);
```

Comment:
When I create Sprites and Animations, I expect file reading and other expensive operations to be done only once. The Images class does that for the image. How about having a similar cache for these Json data? Because I certainly going to want to have multiple choppers' flying out there!

Reply:
Done